Makefile

```diff
@@ -1,6 +1,5 @@
 
 DOCKER ?= $(shell command -v docker)
-PANDOC ?= $(shell command -v pandoc)
```

Comment:
On Mon, May 23, 2016 at 08:21:25PM -0700, Qiang Huang wrote:
> Is it necessary to offer such flexibility? Or does pandoc version
> matters a lot?

The flexibility is just “I want to build the docs easily without
installing Docker”.  I don't think pandoc versions matter much for us.

> I don't think we should make the Makefile complex to meet such
> needs, I think people who really want to use their own local pandoc
> can use it directly without Makefile.

Sure, but supporting them in the Makefile isn't all that much more
complex.  See #448, which also makes the Docker fallback case drier
and adds error handling for folks who have neither Docker nor pandoc
installed locally.